### ISSUE TITLE
Specifying format for Custom Type 

### ISSUE THREAD

**Issue by johnbradley** [common-workflow-language/common-workflow-language#532]
Given a custom type like so:
```
class: SchemaDefRequirement
types:
  - name: NamedFilePairType
    type: record
    fields:
      - name: name
        type: string
      - name: file1
        type: File
      - name: file2
        type: File
```
Is there a way to specify what `format` the files need to be?

I don't see any `format` option in the specification for InputRecordSchema/InputRecordField: http://www.commonwl.org/v1.0/CommandLineTool.html#InputRecordSchema

In my particular case both files are FASTQ (http://edamontology.org/format_1930).

I know I can specify format in a workflow as an input parameter, but this is specifying the format at the wrong level. Example:
```
   # Named read pairs in FASTQ format
   read_pairs:
       type: ../types.yml#NamedFilePairType[]
       format: http://edamontology.org/format_1930 # FASTQ format
```
 For custom types with different format files this would just confuse things.

Thanks.

**Comment by mr-c** [comment]
Thank you @johnbradley , this was an oversight that will be fixed in `v1.1`

**Comment by tetron** [comment]
Will be fixed in #636
